.travis.yml

```diff
@@ -1,29 +1,12 @@
-# Config file for automatic testing at travis-ci.org
-# This file will be regenerated if you run travis_pypi_setup.py
-
 language: python
 python:
-  - 3.5
-  - 3.4
-  - 3.3
-  - 2.7
-  - 2.6
-
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -U tox-travis
```

Comment:
Travis already manages a variety of virtual environments for us, so we don't need tox to do that again on top of those. https://docs.travis-ci.com/user/languages/python/#Using-Tox-as-the-Build-Script